woocommerce/woocommerce
# Docs: Update go-vip.net links/images to developer.woo.com

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:





To get the site ready for launch, this PR updates all the WPVIP testing domain links (https://woo-docs-multi-com.go-vip.net/) to the "real" domain (https://developer.woo.com).



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. **Manually check all files**
2. Verify that all links are valid, and that the find/replace worked as expected

You will need to point `developer.woo.com` to VIP via your hosts file to view this PR.



### Changelog entry




<details>

#### Significance




#### Type




#### Message 

#### Comment 

</details>
